azure-devops/projects/pagopa-projects/pagopa-nodo-pagamenti-test.tf

```
@@ -0,0 +1,120 @@
+  pagopa-nodo-dei-pagamenti-test-variables = {
+    cache_version_id = "v1"
+    default_branch   = var.pagopa-nodo-dei-pagamenti-test.repository.branch_name
+  }
+  # global secrets
+  pagopa-nodo-dei-pagamenti-test-variables_secret = {
+
+  }
+  # code_review vars
+  pagopa-nodo-dei-pagamenti-test-variables_code_review = {
+    sonarcloud_service_conn = var.pagopa-nodo-dei-pagamenti-test.pipeline.sonarcloud.service_connection
+    sonarcloud_org          = var.pagopa-nodo-dei-pagamenti-test.pipeline.sonarcloud.org
+    sonarcloud_project_key  = var.pagopa-nodo-dei-pagamenti-test.pipeline.sonarcloud.project_key
+    sonarcloud_project_name = var.pagopa-nodo-dei-pagamenti-test.pipeline.sonarcloud.project_name
+
+    dev_azure_subscription_storage_account = "DEV-PAGOPA-SERVICE-CONN"
+    dev_storage_account_name               = "pagopadnodotestsa"
+    dev_resource_group_azure               = "pagopa-d-nodo-test-rg"
+    dev_test_execution_max_minutes         = 120
+    dev_sas_token                          = module.secrets.values["DEV-NODO-TEST-SAS-TOKEN"].value
+
+    uat_azure_subscription_storage_account = "UAT-PAGOPA-SERVICE-CONN"
+    uat_storage_account_name               = "pagopaunodotestsa"
+    uat_resource_group_azure               = "pagopa-u-nodo-test-rg"
+    uat_test_execution_max_minutes         = 120
+    uat_sas_token                          = module.secrets.values["UAT-NODO-TEST-SAS-TOKEN"].value
+  }
+  # code_review secrets
+  pagopa-nodo-dei-pagamenti-test-variables_secret_code_review = {
+    danger_github_api_token = "skip"
+  }
+  # deploy vars
+  pagopa-nodo-dei-pagamenti-test-variables_deploy = {
+  }
+  # deploy secrets
+  pagopa-nodo-dei-pagamenti-test-variables_secret_deploy = {
+  }
+}
+
+module "pagopa-nodo-dei-pagamenti-test_code_review" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_code_review?ref=v2.0.4"
+  count  = var.pagopa-nodo-dei-pagamenti-test.pipeline.enable_code_review == true ? 1 : 0
+
+  project_id                   = azuredevops_project.project.id
+  repository                   = var.pagopa-nodo-dei-pagamenti-test.repository
+  github_service_connection_id = azuredevops_serviceendpoint_github.io-azure-devops-github-rw.id
+
+  pull_request_trigger_use_yaml = true
+
+  variables = merge(
+    local.pagopa-nodo-dei-pagamenti-test-variables,
+    local.pagopa-nodo-dei-pagamenti-test-variables_code_review,
+  )
+
+  variables_secret = merge(
+    local.pagopa-nodo-dei-pagamenti-test-variables_secret,
+    local.pagopa-nodo-dei-pagamenti-test-variables_secret_code_review,
+  )
+
+  service_connection_ids_authorization = [
+    azuredevops_serviceendpoint_github.io-azure-devops-github-ro.id,
+    local.azuredevops_serviceendpoint_sonarcloud_id,
+  ]
+}
+
+module "pagopa-nodo-dei-pagamenti-test_deploy" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_deploy?ref=v2.0.4"
+  count  = 0
```

```suggestion
  count  = var.pagopa-nodo-dei-pagamenti-test.pipeline. enable_deploy == true ? 1 : 0
```
use enable/disable flag